src/mapml/control/ScaleBar.js

```diff
@@ -0,0 +1,95 @@
+export var ScaleBar = L.Control.Scale.extend({
+    options: {
+      maxWidth: 100,
+      updateWhenIdle: true,
+      position: 'bottomleft'
+    },
+
+    _container: null,
```

Comment:
A way to initialize stuff is in a constructor, which in leaflet is always called `initialize`.  You can also do a setOptions in there to merge the options you set above with the options that may get passed to `initialize`: https://leafletjs.com/examples/extending/extending-1-classes.html#lclassinitialize

Reply:
I tried removing your _container: null member and replacing it with the equivalent inside a constructor, but the scalebar started showing both imperial and metric at the same time again, so there's something else that I'm missing here.